src/universal/config/routes.ts

```diff
@@ -123,35 +117,36 @@ export const PublicRoutes = [
 ];
 
 export const PrivateRoutes = Object.values(AppRoutes).filter(
-  (path) => !PublicRoutes.includes(path)
+  (path: string) => !PublicRoutes.includes(path)
 );
 
-type AppRoute = keyof typeof AppRoutes;
+type RouteKey = keyof typeof AppRoutes;
+export type AppRoute = (typeof AppRoutes)[RouteKey];
```

Comment:
Door van de variabele `AppRoutes` als type `const` aan te duiden krijgen we in het type `AppRoute` de daadwerkelijke routes in plaats van `string`.